Panic during SSH install on Windows

As observed by @rdrsss

```text
Open http://localhost:5688 in your browser to continue.
INFO[03-02|17:45:48] Event: new_cluster:
INFO[03-02|17:45:48] Event: log: Detecting authentication
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x1 addr=0x0 pc=0x5741c3]

goroutine 43 [running]:
github.com/flynn/flynn/installer.(*SSHCluster).findSSHAuth(0x133a4280, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/installer/ssh_cluster.go:351 +0xe83
github.com/flynn/flynn/installer.*SSHCluster.(github.com/flynn/flynn/installer.findSSHAuth)┬╖fm(0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/installer/ssh_cluster.go:102 +0x37
github.com/flynn/flynn/installer.func┬╖029()
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/installer/ssh_cluster.go:111 +0x2f7
created by github.com/flynn/flynn/installer.(*SSHCluster).Run
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/installer/ssh_cluster.go:125 +0x76

goroutine 1 [IO wait, 2 minutes]:
net.(*pollDesc).Wait(0x132d86f0, 0x72, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_poll_runtime.go:84 +0x42
net.(*ioSrv).ExecIO(0x133231d0, 0x132d8658, 0xac7288, 0x8, 0x133222e8, 0x132d9260, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_windows.go:188 +0x265
net.(*netFD).acceptOne(0x132d8620, 0x132d9180, 0x2, 0x2, 0x132d8658, 0xcdeb08, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_windows.go:558 +0x3ee
net.(*netFD).accept(0x132d8620, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_windows.go:585 +0x119
net.(*TCPListener).AcceptTCP(0x133231e0, 0x13405cb0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/tcpsock_posix.go:234 +0x48
net.(*TCPListener).Accept(0x133231e0, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/tcpsock_posix.go:244 +0x48
net/http.(*Server).Serve(0x13354c00, 0x1a3668, 0x133231e0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1728 +0x7b
net/http.Serve(0x1a3668, 0x133231e0, 0x1a4760, 0x13389010, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1606 +0x8c
github.com/flynn/flynn/installer.ServeHTTP(0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/installer/http.go:107 +0xaf1
main.runInstaller(0x13352fa0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/cli/install.go:21 +0x31
main.runCommand(0x132b2110, 0x7, 0x1325f70, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/cli/main.go:174 +0x621
main.main()
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/cli/main.go:124 +0xac7

goroutine 5 [syscall, 4 minutes]:
os/signal.loop()
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/os/signal/signal_unix.go:21 +0x21
created by os/signal.init┬╖1
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/os/signal/signal_unix.go:27 +0x34

goroutine 11 [chan receive, 4 minutes]:
github.com/flynn/flynn/pkg/shutdown.(*handler).wait(0x1334e520)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/pkg/shutdown/handler.go:58 +0x11d
created by github.com/flynn/flynn/pkg/shutdown.newHandler
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/pkg/shutdown/handler.go:25 +0x89

goroutine 12 [chan receive, 4 minutes]:
database/sql.(*DB).connectionOpener(0x132be730)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/database/sql/sql.go:589 +0x43
created by database/sql.Open
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/database/sql/sql.go:452 +0x256

goroutine 34 [chan receive, 2 minutes]:
github.com/flynn/flynn/pkg/sse.func┬╖001()
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/pkg/sse/stream.go:60 +0x57
created by github.com/flynn/flynn/pkg/sse.(*Stream).Serve
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/pkg/sse/stream.go:62 +0x27d

goroutine 44 [semacquire]:
sync.(*Mutex).Lock(0x132b6748)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/sync/mutex.go:66 +0xcb
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.(*DB).do(0x132b6740, 0x133da880, 0x1a3400, 0x133da890,
0x0, 0x0, 0x13322168, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/ql.go:2021 +0x8a
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.recordset.Do(0x133da880, 0x1a3400, 0x133da890, 0x0, 0x0
, 0x13322168, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/ql.go:98 +0x62
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.(*recordset).Do(0x133da8e0, 0x13322d00, 0x13322168, 0x0
, 0x0)
        <autogenerated>:49 +0x9a
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.func┬╖004()
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/driver.go:399 +0xa7
created by github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.newdriverRows
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/driver.go:407 +0x142

goroutine 39 [semacquire]:
sync.(*Mutex).Lock(0x132b6748)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/sync/mutex.go:66 +0xcb
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.(*DB).do(0x132b6740, 0x133667c0, 0x1a3400, 0x133667d0,
0x0, 0x0, 0x13322508, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/ql.go:2021 +0x8a
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.recordset.Do(0x133667c0, 0x1a3400, 0x133667d0, 0x0, 0x0
, 0x13322508, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/ql.go:98 +0x62
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.(*recordset).Do(0x133667f0, 0x13322100, 0x13322508, 0x0
, 0x0)
        <autogenerated>:49 +0x9a
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.func┬╖004()
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/driver.go:399 +0xa7
created by github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.newdriverRows
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/driver.go:407 +0x142

goroutine 35 [select]:
reflect.Select(0x133b5f98, 0x3, 0x3, 0x133a4640, 0x0, 0x0, 0x0, 0x7c9401)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/reflect/value.go:1965 +0x1d7
github.com/flynn/flynn/pkg/sse.func┬╖002()
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/pkg/sse/stream.go:83 +0x16c
created by github.com/flynn/flynn/pkg/sse.(*Stream).Serve
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/pkg/sse/stream.go:99 +0x3d2

goroutine 36 [IO wait, 2 minutes]:
net.(*pollDesc).Wait(0x132d8fb0, 0x72, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_poll_runtime.go:84 +0x42
net.(*ioSrv).ExecIO(0x133231d0, 0x132d8f18, 0xabcc48, 0x7, 0xcdd9a4, 0x435765, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_windows.go:188 +0x265
net.(*netFD).Read(0x132d8ee0, 0x133c0000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_windows.go:470 +0x11a
net.(*conn).Read(0x13323d18, 0x133c0000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/net.go:121 +0xba
io.Copy(0x1a4cc8, 0x13322280, 0x1a47c8, 0x13323d18, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/io/io.go:362 +0x184
net/http.func┬╖008()
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:171 +0x66
created by net/http.(*conn).closeNotify
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:177 +0x268

goroutine 45 [runnable]:
github.com/flynn/flynn/installer.(*Subscription).SendEvents(0x1338aac0, 0x133305b0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/installer/events.go:35
created by github.com/flynn/flynn/installer.(*Installer).SendEvent
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/installer/events.go:197 +0x6e1

goroutine 41 [runnable]:
sync.(*Mutex).Lock(0x132b6748)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/sync/mutex.go:66 +0xcb
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.(*DB).do(0x132b6740, 0x13367840, 0x1a3400, 0x13367850,
0x0, 0x0, 0x13322918, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/ql.go:2021 +0x8a
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.recordset.Do(0x13367840, 0x1a3400, 0x13367850, 0x0, 0x0
, 0x13322918, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/ql.go:98 +0x62
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.(*recordset).Do(0x13367870, 0x13322700, 0x13322918, 0x0
, 0x0)
        <autogenerated>:49 +0x9a
github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.func┬╖004()
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/driver.go:399 +0xa7
created by github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql.newdriverRows
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/cznic/ql/driver.go:407 +0x142

goroutine 30 [IO wait]:
net.(*pollDesc).Wait(0x132d9170, 0x72, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_poll_runtime.go:84 +0x42
net.(*ioSrv).ExecIO(0x133231d0, 0x132d90d8, 0xabcc48, 0x7, 0xcdd9a4, 0xe75, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_windows.go:188 +0x265
net.(*netFD).Read(0x132d90a0, 0x133cb000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_windows.go:470 +0x11a
net.(*conn).Read(0x13322010, 0x133cb000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/net.go:121 +0xba
net/http.(*liveSwitchReader).Read(0x1337c024, 0x133cb000, 0x1000, 0x1000, 0x439a40, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:214 +0x8b
io.(*LimitedReader).Read(0x13366070, 0x133cb000, 0x1000, 0x1000, 0x133bfc28, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/io/io.go:408 +0xda
bufio.(*Reader).fill(0x1336c120)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/bufio/bufio.go:97 +0x15c
bufio.(*Reader).ReadSlice(0x1336c120, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/bufio/bufio.go:295 +0x1ce
bufio.(*Reader).ReadLine(0x1336c120, 0x0, 0x0, 0x0, 0x132d9100, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/bufio/bufio.go:324 +0x5a
net/textproto.(*Reader).readLineSlice(0x1338a060, 0x0, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/textproto/reader.go:55 +0x76
net/textproto.(*Reader).ReadLine(0x1338a060, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/textproto/reader.go:36 +0x48
net/http.ReadRequest(0x1336c120, 0x1336b2d0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/request.go:598 +0xaf
net/http.(*conn).readRequest(0x1337c000, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:586 +0x24b
net/http.(*conn).serve(0x1337c000)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1162 +0x60f
created by net/http.(*Server).Serve
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1751 +0x2ce

goroutine 29 [chan receive, 2 minutes]:
github.com/flynn/flynn/pkg/sse.ServeStream(0x1a4860, 0x1336a770, 0x968720, 0x133a4640, 0x1a2708, 0x13352fb0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/pkg/sse/stream.go:48 +0x7f
github.com/flynn/flynn/installer.(*httpAPI).Events(0x1336d020, 0x1a4860, 0x1336a770, 0x1336a690, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/installer/http.go:249 +0xef
github.com/flynn/flynn/installer.*httpAPI.Events┬╖fm(0x1a4860, 0x1336a770, 0x1336a690, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/installer/http.go:92 +0x57
github.com/flynn/flynn/Godeps/_workspace/src/github.com/julienschmidt/httprouter.(*Router).ServeHTTP(0x13388140, 0x1a486
0, 0x1336a770, 0x1336a690)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/Godeps/_workspace/src/github.com/julienschmidt/httprouter/router.
go:276 +0x11c
github.com/flynn/flynn/pkg/cors.func┬╖001(0x1a4860, 0x1336a770, 0x1336a690)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/pkg/cors/cors.go:143 +0x1db
net/http.HandlerFunc.ServeHTTP(0x13389010, 0x1a4860, 0x1336a770, 0x1336a690)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1265 +0x3b
net/http.serverHandler.ServeHTTP(0x13354c00, 0x1a4860, 0x1336a770, 0x1336a690)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1703 +0x145
net/http.(*conn).serve(0x1337d6e0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1204 +0x9d8
created by net/http.(*Server).Serve
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1751 +0x2ce

goroutine 28 [IO wait, 2 minutes]:
net.(*pollDesc).Wait(0x132d8d10, 0x72, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_poll_runtime.go:84 +0x42
net.(*ioSrv).ExecIO(0x133231d0, 0x132d8c78, 0xabcc48, 0x7, 0xcdd9a4, 0x13384000, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_windows.go:188 +0x265
net.(*netFD).Read(0x132d8c40, 0x1331c000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/fd_windows.go:470 +0x11a
net.(*conn).Read(0x13323a18, 0x1331c000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/net.go:121 +0xba
net/http.(*liveSwitchReader).Read(0x1337d644, 0x1331c000, 0x1000, 0x1000, 0x439a40, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:214 +0x8b
io.(*LimitedReader).Read(0x133890b0, 0x1331c000, 0x1000, 0x1000, 0x133afc28, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/io/io.go:408 +0xda
bufio.(*Reader).fill(0x132b5a70)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/bufio/bufio.go:97 +0x15c
bufio.(*Reader).ReadSlice(0x132b5a70, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/bufio/bufio.go:295 +0x1ce
bufio.(*Reader).ReadLine(0x132b5a70, 0x0, 0x0, 0x0, 0x132d8c00, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/bufio/bufio.go:324 +0x5a
net/textproto.(*Reader).readLineSlice(0x1338a080, 0x0, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/textproto/reader.go:55 +0x76
net/textproto.(*Reader).ReadLine(0x1338a080, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/textproto/reader.go:36 +0x48
net/http.ReadRequest(0x132b5a70, 0x1336a380, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/request.go:598 +0xaf
net/http.(*conn).readRequest(0x1337d620, 0x0, 0x0, 0x0)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:586 +0x24b
net/http.(*conn).serve(0x1337d620)
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1162 +0x60f
created by net/http.(*Server).Serve
        /tmp/tmp.nvZjhkv9rt/src/github.com/flynn/flynn/util/_toolchain/go/src/net/http/server.go:1751 +0x2ce
```

--- a/installer/ssh_cluster.go
+++ b/installer/ssh_cluster.go
@@ -348,9 +348,11 @@ func (c *SSHCluster) findSSHAuth() error {
 	}
 
 	sshAgent := c.sshAgent()
-	sshAgentAuth := ssh.PublicKeysCallback(sshAgent.Signers)
-	for _, t := range c.Targets {
-		testAndAddAuthMethod(t, sshAgentAuth)
+	if sshAgent != nil {
+		sshAgentAuth := ssh.PublicKeysCallback(sshAgent.Signers)
+		for _, t := range c.Targets {
+			testAndAddAuthMethod(t, sshAgentAuth)
+		}
 	}
 
 	if testAllAuthenticated(c.Targets) {
@@ -358,10 +360,12 @@ func (c *SSHCluster) findSSHAuth() error {
 	}
 
 	var agentKeys [][]byte
-	if keys, err := sshAgent.List(); err == nil {
-		agentKeys = make([][]byte, len(keys))
-		for i, k := range keys {
-			agentKeys[i] = k.Marshal()
+	if sshAgent != nil {
+		if keys, err := sshAgent.List(); err == nil {
+			agentKeys = make([][]byte, len(keys))
+			for i, k := range keys {
+				agentKeys[i] = k.Marshal()
+			}
 		}
 	}
 

